src/status_im/ui/screens/browser/views.cljs

```diff
@@ -115,16 +115,16 @@
        [react/view styles/background
         [react/text (i18n/label :t/enter-dapp-url)]])
      [react/view styles/toolbar
-      [react/touchable-highlight {:on-press            #(.goBack @webview)
-                                  :disabled            (not can-go-back?)
+      [react/touchable-highlight {:on-press            #(browser-history/back browser)
+                                  :disabled            (not (browser-history/can-go-back? browser))
                                   :accessibility-label :previou-page-button}
-       [react/view (when (not can-go-back?) {:opacity 0.4})
+       [react/view (when (not (browser-history/can-go-back? browser)) {:opacity 0.4})
```

Comment:
you can move these calls to let , to have two vars for example, `back-disabled?` and `forward-disabled?` to do not call twice here and code will be more readable , thanks

Reply:
Don't think that's possible: the let only gets invoked when the browser is initially opened, so that won't work. :)